test/unit-test/ota_utest.c

```diff
@@ -972,6 +972,17 @@ void test_OTA_InitNullAppBuffers()
     TEST_ASSERT_EQUAL( OtaAgentStateInit, OTA_GetState() );
 }
 
+void test_OTA_InitNullOtaAppPointer()
+{
+    OtaAppBuffer_t * otaAppBuffer = NULL;
+
+    OTA_Init( otaAppBuffer,
```

Comment:
Return value of OTA_Init should be checked ( Should be OtaErrInvalidArg right? ).

Reply:
Based on the current implementation of the function `OTA_Init` only returns two values. If there are no errors during the initialization of the agent then it returns `OtaErrNone` and `OtaErrUninitialized` otherwise.